Cargo.toml

```diff
@@ -36,6 +42,10 @@ crossterm = "0.19"
 fancy-regex = "0.4"
 signal-hook = "0.3"
 rayon = "1.5"
+#nlprule = { git = "https://github.com/bminixhofer/nlprule.git", optional = true }
+nlprule = { git = "https://github.com/drahnr/nlprule.git", optional = true }
```

Comment:
Do you wanna keep the commented versions? And why do you choose your fork?

Reply:
That was pre merge if my PR there :)